core/dbt/adapters/base/impl.py

```diff
@@ -551,8 +517,7 @@ def list_relations_without_caching(
         :rtype: List[self.Relation]
         """
         raise NotImplementedException(
-            '`list_relations_without_caching` is not implemented for this '
-            'adapter!'
+            "`list_relations_without_caching` is not implemented for this " "adapter!"
```

Comment:
This is kind of weird. I guess it works... but seems like something we should cleanup at some point.

Reply:
I like that it's making lines fit the configured length when they can

Reply:
What's weird about it?  Looks like a straightforward application of 
`black uses single quotes` + `black honors line length` to me.  

The original code is a little weird (two quoted strings concat'd by the fact they follow each other), but that's not black's fault.